router/src/lib.rs

```diff
@@ -15,6 +15,64 @@ use tracing::warn;
 use utoipa::ToSchema;
 use validation::Validation;
 
+#[cfg(feature = "kserve")]
+mod kserve {
```

Comment:
Let's put this in its own file, no ?

Reply:
all kserve types and routes moved into new file